src/pages/CurrenciesListPage.jsx

```diff
@@ -13,20 +14,28 @@ import {
   Td,
 } from "../components/Table";
 import "../styles/CurrenciesListPage.scss";
+import useCurrenciesList from "../components/useCurrenciesListHook";
+import Loading from "../components/Loading";
 
 function CurrenciesList() {
-  const options = [
-    { currency: "elo", label: "elo" },
-    { currency: "siema", label: "siema" },
-  ];
+  const { list, loading, error } = useCurrenciesList();
+  if (error) {
+    const e = new Error("Could not load the page");
+    throw e;
+  }
```

Comment:
Do not throw an error, just render a user-friendly message on the screen in case an error occurred, we don't want the application to just crash randomly for the user

Reply:
it makes much more sense xD